### ISSUE TITLE
Can not download English bin version

### ISSUE THREAD

**Issue by JLenssen** [facebookresearch/fastText#727]
I have the same problem as in : https://github.com/facebookresearch/fastText/issues/620
for the English binary model. I am able to download the French and German binary model but not the English one.

The error is: 
```
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>085D472F3639923B</RequestId>
<HostId>
oqLuxRqTEYFP/XZR+hh41iTUj6dfdcJcVLMhs39gzE76Buala6A2zu6qYRuaKG4cIy0MycVEpw8=
</HostId>
</Error>
```

Is there a way to download the English binary models somewhere else at the moment?

**Comment by piotr-bojanowski** [comment]
Hi JLenssen,

There is a typo in the URL on the webpage. We will fix that very shortly but in the meantime, you can download the vectors at:
https://dl.fbaipublicfiles.com/fasttext/vectors-crawl/cc.en.300.bin.gz
https://dl.fbaipublicfiles.com/fasttext/vectors-crawl/cc.en.300.vec.gz

Best,

Piotr

**Comment by piotr-bojanowski** [comment]
Thanks a lot @JLenssen for raising this issue!

This bug was fixed by commit bc1285939f1c216bd358425c3685a049dd8f56c0.

Best regards,

Piotr
